.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Before filing a bug please check the following: -->
```

Comment:
Do you want this list in plain text or keep it in HTML comments?

Reply:
In HTML comments, reporters may fill the information, without knowing that this won't show up in the final rendered ticket if they don't remove the comments.

Reply:
It can be kept in  comment. It's not supposed to be edited by reporters.

Reply:
List is kept in comments.